### PR TITLE
1178: Edits to fingerprinting.rst

### LINKED ISSUE #1178: DOC: edits to fingerprinting.rst
Edits to fingerprinting.rst

### PR DESCRIPTION
**Pull request by jlprevatt**
Please review the changes. Thanks.

Closes #1178

### CODE REVIEW

File: docs/source/fingerprinting.rst [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
 
-System Fingerprints
+Fingerprinting is the result of Quipucords analytical processing that occurs when you request a summary report for a scan. If you request a detailed report, you receive the raw facts for that scan without any fingerprinting. When you request a summary report, the deduplication, merging, and post-processing of facts occurs to produce the fingerprint. These processes include identifying installed products and versions from the raw facts, finding consumed entitlements, finding and merging duplicate instances of products from different sources, and finding products installed in nondefault locations, among other steps.
```
Comment (chambridge) [inline docs/source/fingerprinting.rst]: Fingerprinting actually occurs the moment the scan results are available, not when you request a summary report. The resulting fingerprints are stored in the database and retrieved when a summary report is requested.

Comment (chambridge) [reply docs/source/fingerprinting.rst]: Not sure? nondefault vs. non-default?

Comment (jlprevatt) [reply docs/source/fingerprinting.rst]: RE the non-default vs. nondefault question. RH CCS uses IBM Style as its style guide. IBM style says no hyphens except in some special cases if the dictionary specifies or for clarity--e.g., recreate (play) vs. re-create (create again). So the default is no hyphen.

Comment (chambridge) [reply docs/source/fingerprinting.rst]: ok 👌

Comment (jlprevatt) [reply docs/source/fingerprinting.rst]: RE the comment on fingerprinting and when it occurs: Reworked some of the info in that paragraph per the comments.

**Pull request merged**